mosip/on-prem/README.md

```diff
@@ -161,8 +161,14 @@ kubectl apply -f https://rancher.e2e.mosip.net/v3/import/pdmkx6b4xxtpcd699gzwdtt
 * Wait for few seconds after executing the command for the cluster to get verified.
 * Your cluster is now added to the rancher management server.
 
-## Longhorn
-* Install [Longhorn](../../longhorn/README.md) for persistent storage.
+## The following storage classes can be used:
+
+* [Vsphere storage class](https://github.com/vmware-archive/vsphere-storage-for-kubernetes): If you are already using VMware virtual machines, you can proceed with the vSphere storage class.
+* [NFS client provisioner storage class](../../nfs/README.md).
+* [ceph-csi](TODO Implementation in progress)
+* [Longhorn](../../longhorn/README.md)
+
+For time being considering Longhorn as a storage class.
```

Comment:
For time being considering Longhorn as a storage class. - This sentence is incorrect... For time being. let us consider Longhorn as a storage class. 9or pls check the technical context and update accordingly